helm/defectdojo/templates/_helpers.tpl

```diff
@@ -166,7 +166,7 @@ Create chart name and version as used by the chart label.
           key: postgresql-postgres-password
         {{- end }}
   {{- if .Values.extraEnv }}
-  {{- toYaml .Values.extraEnv | nindent 8 }}
+  {{- toYaml .Values.extraEnv | nindent 2 }}
   {{- end }}
   resources:
     {{- toYaml .Values.django.uwsgi.resources | nindent 10 }}
```

Comment:
@DevSecOps-Isotrol would you adjust this indentation as well?
With `nindent 10`:
```
        resources:
                limits:
                  cpu: 2000m
                  memory: 512Mi
                requests:
                  cpu: 100m
                  memory: 256Mi
```
With `nindent 4`:
```
        resources:
          limits:
            cpu: 2000m
            memory: 512Mi
          requests:
            cpu: 100m
            memory: 256Mi
```
I know it doesn't break the chart, but probably this leaded to the error you're fixing in this PR.

Reply:
Solved:

```yaml
        resources:
          limits:
            cpu: 2000m
            memory: 512Mi
          requests:
            cpu: 100m
            memory: 256Mi

```